cmd/kube-proxy/app/server.go

```diff
@@ -924,16 +924,16 @@ func (s *ProxyServer) Run() error {
 	// Note: RegisterHandler() calls need to happen before creation of Sources because sources
 	// only notify on changes, and the initial update (on process start) may be lost if no handlers
 	// are registered yet.
-	serviceConfig := config.NewServiceConfig(s.logger, informerFactory.Core().V1().Services(), s.Config.ConfigSyncPeriod.Duration)
+	serviceConfig := config.NewServiceConfig(context.TODO(), informerFactory.Core().V1().Services(), s.Config.ConfigSyncPeriod.Duration)
```

Comment:
they can't just be `context.TODO()` though; you need a context that includes `s.logger`